### DISCUSSION TITLE
Use mapTypeId:hybrid with mapId managed in cloud console does not show set styles.

### DISCUSSION THREAD

**Discussion by nw-kohei-ito** [visgl/react-google-maps#397] [Q&A]
Hello I am trying to use  react library "@vis.gl/react-google-maps" with vector map mapTypeId: "hybrid". I set syle and make that style public. When I use mapTypeId: "satellite" or "roadmap", stype applied properly. But with "hybrid" tyle is like default. Why is that? 

Is there any way to apply style that is connected with mapId in google map console when I use mapTypeId: "hybrid"?

**Comment by usefulthink** [comment]
I believe that is a limitation / bug in the Maps API itself. 
This is tracked in the public issue-tracker here (sadly, though, it's marked as "Won't fix (intended behavior)"): https://issuetracker.google.com/issues/180554122
Maybe adding stars and comments on that issue will at some point make them change their mind...

What might work as a workaround is to use the raster rendering-mode and the local styling, but I don't know if that's actually the case and even if, that might not be feasible for you.

**Reply by nw-kohei-ito** [reply]
Thank you for your answer; that is good information for me because I can find an alternative solution. I have upvoted the issue you mentioned. I hope they change their mind too. Thanks!
